types/index.d.ts

```diff
@@ -16,7 +16,7 @@ export * from './lib/errors';
 export { BaseError as Error } from './lib/errors';
 export { useInflection } from './lib/utils';
 export { Utils, QueryTypes, Op, TableHints, IndexHints, DataTypes, Deferrable };
-export { Validator as validator } from './lib/utils/validator-extras';
+export { Validator } from './lib/utils/validator-extras';
```

Comment:
Could this be seen as a breaking change for some people?

Reply:
Also, shouldn't there be a test which should be updated for this?

Reply:
Oh yes, good point. I'll include the old name as well, but deprecated.

As for tests, I'm not sure how we can test that the typings match the JS file. It should not be an issue once the typescript migration is complete though.

Reply:
Ah, if it's undefined now it should not be a problem.

For the tests; I also wouldn't really know to be honest. Not very experienced on that front. Should also indeed be fixed with the TypeScript migration so not a huge priority.

Reply:
https://github.com/sequelize/sequelize/pull/13689 includes a test that ensures no declared export has undefined as a value. I think that + the typescript migration is the best we can